src/MobileDelivery/MobileComplete.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import moment from 'moment';
 import './Mobile.css';
+import MobileConfirm from './MobileConfirm';
 
 class MobileComplete extends React.Component {
     render() { 
```

Comment:
same nit-picks about not your changes..:
- line 18 & 23: format from timestamp moment
- line 50: "Pick Up From" -> "Deliver To"